js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetNode.tsx

```diff
@@ -46,7 +46,9 @@ export const AssetNode = React.memo(({definition, selected, onChangeAssetSelecti
           flex={{direction: 'row', justifyContent: 'space-between', alignItems: 'center'}}
           style={{minHeight: ASSET_NODE_TAGS_HEIGHT, marginTop: marginTopForCenteringNode}}
         >
-          <StaleReasonsTag liveData={liveData} assetKey={definition.assetKey} />
+          <div>
```

Comment:
@bengotow Opted to just wrap the `StaleReasonTag` with a div instead of making `StaleReasonTag` return an empty div because that was causing all of the other spots using `StaleReasonTag` to end up with weird spacing because they would wrap it with other tags using a flex gap and the empty div caused an extra gap.

Reply:
Hmm I think this is fine here because there is no `gap` between the flex items 🫡